src/analysis/retail/monk/mistweaver/modules/spells/EssenceFontTargetsHit.tsx

```diff
@@ -35,6 +37,17 @@ class EssenceFontTargetsHit extends Analyzer {
     this.targetsEF += 1;
   }
 
+  getPerformance(targetsHit: number) {
+    if (targetsHit < 12) {
+      return QualitativePerformance.Fail;
+    } else if (targetsHit < 14) {
```

Comment:
Average targets hit per cast is rarely above is 14, I would bump these down.
This also doesn't account for Upwelling bolts, you could do something like make these values a percentage of expected bolts - which is already calc'ed in cancelled casts